2019/CFP.html

```diff
@@ -7,14 +7,14 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="description" content="Conference project">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<link rel="stylesheet" type="text/css" href="../styles/bootstrap4/bootstrap.min.css">
+	<link rel="stylesheet" type="text/css" href="styles/bootstrap4/bootstrap.min.css">
```

Comment:
Just for clarification - this will enable each legacy 'site' to keep all of it's styling separate - is that the intended goal?
So if we make updates to the main styles, the old sites won't get them?

While I understand the desire to maintain the old content, maybe it would make sense to add a single "legacy" styling folder instead? That way all previous sites can use one set style instead of having to maintain a separate styling folder for each.

Reply:
Yes, that was the intention, and I can see how just having a single legacy folder for the old styles would be better. I'll make that change and commit to this branch.

Reply:
@c-ryan-k the old styles are now in a common legacy folder.